### PR TITLE
Add time support

### PR DESCRIPTION
**Pull request by bffcorreia**

### CODE REVIEW

File: src/main/java/io/bloco/faker/Faker.java [hunk 1]
```diff
@@ -52,6 +52,8 @@ public Faker(String locale) {
         this.book = this.data.getComponent(Book.class);
         this.company = this.data.getComponent(Company.class);
         this.name = this.data.getComponent(Name.class);
+        // TODO: CHANGE THIS??
+        this.time = new Time();
```
Comment (sdsantos) [inline src/main/java/io/bloco/faker/Faker.java]: I instantiated the componentes in the FakerData constructor. Then you can use this.data.getComponent to find it.

File: src/main/java/io/bloco/faker/components/Time.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
+package io.bloco.faker.components;
+
+import io.bloco.faker.helpers.Period;
+import java.util.Date;
+import java.util.Random;
+import org.joda.time.DateTime;
+
+public class Time {
```
Comment (sdsantos) [inline src/main/java/io/bloco/faker/components/Time.java]: Each component is extending FakerComponent.

File: src/main/java/io/bloco/faker/components/Time.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+  private final Random random;
+
+  public Time() {
+    this.random = new Random();
```
Comment (sdsantos) [inline src/main/java/io/bloco/faker/components/Time.java]: I'm using the RandomHelper in FakerComponent for random stuff. You can make it protected and use it here.

File: src/main/java/io/bloco/faker/helpers/Period.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23
+  }),
+  day(new int[] { 9, 10, 11, 12, 13, 14, 15, 16, 17 }),
+  night(new int[] { 18, 19, 20, 21, 22, 23 }),
+  morning(new int[] { 6, 7, 8, 9, 10, 11 }),
+  afternoon(new int[] { 12, 13, 14, 15, 16, 17 }),
+  evening(new int[] { 17, 18, 19, 20, 21 }),
+  midnight(new int[] { 0, 1, 2, 3, 4 });
+
+  private int[] values;
```
Comment (sdsantos) [inline src/main/java/io/bloco/faker/helpers/Period.java]: This could be final :)

**Comment by sdsantos** [comment]
Added some comments @bffcorreia, but it looks good, just minor changes ;)

**Comment by bffcorreia** [comment]
@sdsantos I made some small changes based on your comments :) Pls check my last commit.

**Pull request merged**